MAUI/Picker/events.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: Events with .NET MAUI Picker control | Syncfusion
+description: Learn here all about events with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Events in .NET MAUI Picker (SfPicker)
+
+Three events have been used for a picker when it is in the Dialog mode. They are, 
+
+ * Opened 
+ * Closing 
+ * Closed
+
+## Opened
+
+The `Opened` event occurs when the picker pop-up is opened.
+
+## Closing event
+
+The `Closing` event raises when the picker pop-up gets closing. You can stop the picker pop-up close action by setting the e.cancel to true.
```

Comment:
The "Closing" event is raised when the picker pop-up is closing. You can prevent the picker pop-up from closing by setting "e.cancel" to true.